lib/Doctrine/DBAL/Types/ConversionException.php

```diff
@@ -36,14 +36,19 @@ class ConversionException extends \Doctrine\DBAL\DBALException
      *
      * @param string $value
      * @param string $toType
+     * @param \Exception|null $previous
      *
      * @return \Doctrine\DBAL\Types\ConversionException
      */
-    static public function conversionFailed($value, $toType)
+    static public function conversionFailed($value, $toType, \Exception $previous = null)
```

Comment:
@greg0ire yeh, that's what I'll do if this solution works for you. It would indeed be different if these exception classes were `final`